py/desisim/test/test_top_level.py

```diff
@@ -8,7 +8,7 @@
 #
 import unittest
 import re
-from .. import __version__ as theVersion
+from desisim import __version__ as theVersion
```

Comment:
I don't think you need to make this change.  Please point me to a test failure that necessitates this change.

Reply:
I looked but I can't find the test where this failed, but there was definitely a complain about the relative vs absolute import. I'll revert the change.

Reply:
In Python 3, *all imports are absolute*, even when they are denoted with `..` notation.  I do notice that the test has some `from __future__` imports that are not needed in Python 3.

Reply:
Done in https://github.com/desihub/desisim/pull/559/commits/db2ac280d3b5999ccfbe993bd57e4a752bd88132.